### PR TITLE
Upgrade cider to latest stable version (0.17.0)

### PR DESCRIPTION
**Pull request by mourjo**

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -23,10 +23,19 @@
...
+                          [refactor-nrepl "2.4.0-SNAPSHOT"]]
                 :dependencies [^:replace [org.clojure/tools.nrepl "0.2.12"]]}}
       #+end_src
+    + Additionally, since for the time being, we are using a snapshot
+      version of refactor-nrepl, you might have to set the following
+      environment var:
+      #+BEGIN_SRC shell
+      export LEIN_JVM_OPTS=-Dhttps.protocols=TLSv1.2
+      #+END_SRC
     + You don't need the cider-nrepl or refactor-nrepl plugin section
```
Comment (vedang) [inline README.org:30]: Let's wait a few days for the 2.4.0 release. Would prefer to merge this change in with that. If it doesn't get created upstream by then, then lets drop to 2.3.1

File: hs-clj-packages.el [hunk 2]
```diff
@@ -112,11 +112,11 @@ cider."
...
   '(;; Clojure Interactive Development Environment that Rocks
     (:name cider
-           :checkout "v0.14.0"
+           :checkout "v0.17.0"
            :after (progn (load-cider-config)))
     ;; A collection of simple clojure refactoring functions
     (:name clj-refactor
-           :checkout "2.2.0"
+           :checkout "2.3.1"
            :after (progn (load-clj-refactor-config))))
```
Comment (vedang) [inline hs-clj-packages.el:120]: same here. Let's wait a few days for the 2.4.0 release. Would prefer to merge this change in with that. If it doesn't get created upstream by then, then lets drop to 2.3.1.

Also, ideally the version of this should always match the corresponding version of refactor-nrepl

File: init.el [hunk 3]
```diff
@@ -119,7 +128,8 @@ Ideally, this will be ~/.emacs.d.")
                          (global-set-key (kbd "M-y") 'helm-show-kill-ring)
                          (global-set-key (kbd "C-x c SPC") 'helm-all-mark-rings)
                          (global-set-key (kbd "C-h SPC") 'helm-all-mark-rings)
-                         (global-set-key (kbd "C-x c r i") 'helm-register)))
+                         (global-set-key (kbd "C-x c r i") 'helm-register)
+                         (global-set-key (kbd "M-i") 'helm-imenu)))
          (:name helm-ag
```
Comment (vedang) [inline init.el:133]: I would prefer if this key-binding also lives within the same scope as other `helm-` key-bindings (i.e. `C-x c ...`). Also, I think that binding `helm-semantic-or-imenu` is slightly more useful. Can you change this to `C-x c i` for `helm-semantic-or-imenu` ?

**Review by vedang — COMMENTED** [review]
Hey @mourjo : I've added a few review comments. Can you incorporate those changes into your commits and re-submit the PR?

**Comment by mourjo** [comment]
Thanks @vedang, I'll make the changes and resubmit it.

**Comment by mourjo** [comment]
@vedang I have made changes you suggested, can you check? Thanks!

**Comment by vedang** [comment]
I have pushed changes to upgrade cider to the latest version (0.19.0), closing this PR.

**Pull request closed without merging**